pkg/util/util.go

```diff
@@ -139,11 +139,18 @@ func ConvertToIBytes(value string) string {
 	if value == "" {
 		return value
 	}
-	iBytes, err := units.RAMInBytes(value)
+	var bytes int64
+	var err error
+	if strings.Contains(value, "i") {
+		bytes, err = units.RAMInBytes(value)
+	} else {
+		bytes, err = units.FromHumanSize(value)
+	}
+	// should error be also returned?
 	if err != nil {
 		return value
 	}
-	return units.BytesSize(float64(iBytes))
+	return units.CustomSize("%.1f%s", float64(bytes), 1024.0, []string{"B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"})
```

Comment:
Need to find a better format specifier such that 1024 bytes show up as `1KiB` and not `1.0KiB`.

Reply:
Maybe upto 2 decimal place is a better precision.